recipes/python-liquid/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is already included in the source distribution so it doesn't need to be included here.

Reply:
`LICENCE` has been removed